examples/fetching_messages.py

```diff
@@ -9,10 +9,9 @@
 
 # Make sure you've imported either the mcap-ros1-support or mcap-protobuf-support
 # libraries before making this call in order to get decoded messages.
-messages = client.get_messages(
+for schema, channel, message, decoded_message in client.iter_messages(
```

Comment:
Should we do anything with `decoded_message` in this example?

Reply:
I started to, but doing anything useful seemed dependent on the data and implementation (per comment above). I could prefix with an underscore to indicate it's unused?

Reply:
Makes sense. I think it's fine to leave it as is to show it's there